src/test/java/core/util/HODateTimeTest.java

```diff
@@ -22,57 +21,56 @@ class HODateTimeTest {
 
     private static final HODateTime HO_DATE_TIME = HODateTime.fromHT("2024-01-01 00:00:00");
 
-        @Test
-        void test() {
-            var nextTraining = HODateTime.fromHT("2022-03-31 08:30:00");
-            var localDateTime = nextTraining.toLocaleDateTime();
-            var previousTraining = nextTraining.plusDaysAtSameLocalTime(-7);
-            var localPrevious = previousTraining.toLocaleDateTime();
-
-            var fetchedDate = HODateTime.fromHT("2022-01-08 14:33:58");
+    @Test
+    void test() {
+        var nextTraining = HODateTime.fromHT("2022-03-31 08:30:00");
+        var localDateTime = nextTraining.toLocaleDateTime();
+        var previousTraining = nextTraining.plusDaysAtSameLocalTime(-7);
+        var localPrevious = previousTraining.toLocaleDateTime();
 
-            Assertions.assertEquals("2022-01-08 14:33:58", fetchedDate.toHT());
+        var fetchedDate = HODateTime.fromHT("2022-01-08 14:33:58");
 
-//            Assertions.assertEquals("08.01.2022", fetchedDate.toLocaleDate());
-//            Assertions.assertEquals("08.01.2022, 14:33:58", fetchedDate.toLocaleDateTime());
+        assertThat("2022-01-08 14:33:58").isEqualTo(fetchedDate.toHT());
 
-            var ts = fetchedDate.toDbTimestamp();
-            Assertions.assertEquals("2022-01-08 14:33:58", HODateTime.fromDbTimestamp(ts).toHT());
+//            assertThat("08.01.2022").isEqualTo(fetchedDate.toLocaleDate());
```

Comment:
I see this was already commented out, remove if there is no value – although curious why it was commented out in the first place.

Reply:
Fixed